docs/about/products.md

```diff
@@ -0,0 +1,72 @@
+The [Made with Unity](http://madewith.unity.com) site showcases published Unity games. These games demonstrate the breadth of possibility using Unity.
+
+Unity has traditionally used the Mono runtime to execute "C# scripts". More recently, Unity has been adopting its [IL2CPP](http://blogs.unity3d.com/2014/05/20/the-future-of-scripting-in-unity/) technology.
+
+.NET Native
+-----------
+
+.NET Native is a set of native build tools for .NET Core. .NET Native is an Ahead-of-Time (AOT) toolchain that produces native applications by compiling IL byte code to native machine code, . This means that the resulting binary is what the OS executes; there is no JIT-ing, no runtime compilation. This leads to better startup performance, as well as some security benefits.
+
+.NET Native is primarily used by .NET [Universal Windows Platform (UWP)](https://msdn.microsoft.com/library/windows/apps/dn726767.aspx) applications.
+
+Windows 10 IoT Core
+-------------------
+
+[Windows 10 IoT Core](https://developer.microsoft.com/windows/iot) is a version of Windows 10 that is optimized for smaller devices with or without a display, and that runs on the Raspberry Pi 2 and 3, Arrow DragonBoard 410c & MinnowBoard MAX. Windows 10 IoT Core utilizes the rich, extensible Universal Windows Platform (UWP) API for building great solutions.
+
+You can build IoT apps in C# using .NET APIs using Visual Studio, with rich debugging of your app running on-device. You can see [examples of IoT apps](https://www.hackster.io/windowsiot) that have been built.
+
+.NET Core
+---------
+
+.NET Core is a general purpose and cross-platform implementation of the .NET Platform. [ASP.NET Core](https://github.com/aspnet/home) is the primary workload that runs on .NET Core.
+
+The following are the main characteristics of .NET Core:
+
+**Cross-platform** - .NET Core supports three operating systems families: Linux, Windows and macOS. .NET Core apps are cross-platform by default. You need to build and publish a version per OS you want to target. The app host and the runtime are the primary OS-specific native dependencies. .NET Core libraries can run unmodified across supported OSes.
+
+**Open Source** - [.NET Core](https://github.com/dotnet/core) is available on GitHub, licensed with the [MIT](https://github.com/dotnet/coreclr/blob/master/LICENSE.TXT) and [Apache 2](https://github.com/dotnet/roslyn/blob/master/License.txt) licenses (licensing is per component). It also makes use of a significant set of open source industry [dependencies](https://github.com/dotnet/core/releases) (see release notes). 
+
+**Natural acquisition** - .NET Core is distributed as a set of packages that developers can pick and choose from. The runtime and base framework can be acquired from [NuGet](http://www.nuget.org/) and OS-specific package managers, such as APT, Homebrew and Yum. Docker images are available on docker hub. The higher-level framework libraries and the larger .NET library ecosystem are available on NuGet. 
```

Comment:
This paragraph didn't seem as clear. How about this:

**Natural acquisition** - .NET Core is distributed as a set of packages. Developers pick and choose from that set based on need. The runtime and base framework are acquired from [NuGet](http://www.nuget.org) and other OS-specific package managers, such as APT (link), Homebrew (link) and Yum (link). Docker images including .NET Core are available on docker hub (link). Developers can add other packages as needed from the rich .NET library ecosystem available on NuGet."